### PR TITLE
To be merged for 7.1 MTA-4014: Custom rules in analysis Dependency rule is not fired

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [MTA-4014](https://issues.redhat.com/browse/MTA-4014)

    * [MTA-4016](https://issues.redhat.com/browse/MTA-4016)

### PREVIEW

* [7.1.1 Known issues](https://anarnold97.github.io/MTA-Preview/MTA/MTA-4014.html#known-issues-7-1-1_release-notes)

* [7.1.0 Known issues](https://anarnold97.github.io/MTA-Preview/MTA/MTA-4014.html#known-issues-7-1-0_release-notes)

### CODE REVIEW

File: docs/topics/analyzer-rbac-known-issue-snippet.adoc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// snippet
+
+.No multi-user access restrictions on resources
+
+There are no multi-user access restrictions on resources. For example, an analyzer task created by a user, can be canceled by any other user. link:https://issues.redhat.com/browse/MTA-4016[(MTA-4016)]
```
Comment (mguetta1) [inline docs/topics/analyzer-rbac-known-issue-snippet.adoc:5]: Shouldn't it be linked to the Jira bug? https://issues.redhat.com/browse/MTA-3819

File: docs/topics/analysis-dependency-rule-snippet.adoc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// snippet
+
+.Dependency rule of custom rules in analysis is not fired
+
+The dependency rule, part of the custom rules, is not fired, and no related migration issue is found. link:https://issues.redhat.com/browse/MTA-4014[(MTA-4014)]
```
Comment (mguetta1) [inline docs/topics/analysis-dependency-rule-snippet.adoc:5]: 1. A custom rules file doesn't have to include a dependency rule.
2. Jira bug: https://issues.redhat.com/browse/MTA-3863

```suggestion
.Dependency rule of a custom rules file in analysis is not fired

A custom dependency rule is not fired, and no related migration issue is found. link:https://issues.redhat.com/browse/MTA-3863[(MTA-3863)]
```

**Review by mguetta1 — CHANGES_REQUESTED** [review]

**Comment by anarnold97** [comment]
As these issues will not be resolved until after 7.2.0, I have copied the two snippets over to the draft 7.2.0 release notes.... as I known @mguetta1 will ask!  :)

File: docs/topics/analyzer-rbac-known-issue-snippet.adoc (hunk 1 above)
Comment (istein1) [reply docs/topics/analyzer-rbac-known-issue-snippet.adoc]: Both snippets LGTM.

**Review by mguetta1 — APPROVED** [review]
LGTM. Thanks!

**Review by dymurray — APPROVED** [review]

File: docs/topics/analyzer-rbac-known-issue-snippet.adoc [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// snippet
+
+.No multi-user access restrictions on resources
+
+There are no multi-user access restrictions on resources. For example, an analyzer task created by a user, can be canceled by any other user. link:https://issues.redhat.com/browse/MTA-3819[(MTA-3819)]
```
Comment (anarnold97) [inline docs/topics/analyzer-rbac-known-issue-snippet.adoc:5]:
```suggestion
There are no multi-user access restrictions on resources. For example, an analyzer task created by a user can be canceled by any other user. link:https://issues.redhat.com/browse/MTA-3819[(MTA-3819)]
```

**Pull request merged**